lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1511,9 +1507,10 @@ void SPIRVToLLVM::transGeneratorMD() {
 
 Value *SPIRVToLLVM::oclTransConstantSampler(SPIRV::SPIRVConstantSampler *BCS,
                                             BasicBlock *BB) {
-  auto *SamplerT =
-      getOrCreateOpaquePtrType(M, OCLOpaqueTypeOpCodeMap::rmap(OpTypeSampler),
-                               getOCLOpaqueTypeAddrSpace(BCS->getOpCode()));
+  std::string Name = kSPIRVTypeName::PrefixAndDelim +
```

Comment:
NIT: could we encapsulate this logic into some function like `getSPIRVOpaquePtrType` to reduce duplicated code a litte bit?

```cpp
PointerType *getSPIRVOpaquePtrType(Module *M, Op OC) {
  std::string Name = getSPIRVTypeName(SPIRVOpaqueTypeOpCodeMap::rmap(OC));
  return getOrCreateOpaquePtrType(M, Name, getOCLOpaqueTypeAddrSpace(OC));
}
```

Reply:
Thank you, done.